### PR TITLE
[MM-52121]: Migrate About build modal unit test to testing library

### PR DESCRIPTION
**Pull request by AshishDhama**
#### Summary
Migrate About build modal unit test to testing library

#### Ticket Link
 Fixes https://mattermost.atlassian.net/browse/MM-52121
  
#### Screenshots
NA

#### Release Note
```release-note

Migrate About build modal unit test to testing library

```

### CODE REVIEW

File: webapp/channels/package.json [hunk 1]
```diff
@@ -116,6 +116,7 @@
     "@redux-devtools/extension": "3.2.3",
     "@testing-library/jest-dom": "5.16.4",
     "@testing-library/react": "12.1.4",
+    "@testing-library/user-event": "12.1.4",
```
Comment (M-ZubairAhmed) [inline webapp/channels/package.json:119]: Why did this addition not generate new changes in a lock file?

Comment (AshishDhama) [reply webapp/channels/package.json:119]: I think package lock already have it, as boards is using it.

**Review by M-ZubairAhmed — APPROVED** [review]

**Pull request merged**